scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -552,6 +554,14 @@ protected void buildCellContextualMenu(Menu menu) {
         controller.getJobSignals(ids.get(0), this);
         controller.checkJobsPermissionMethods(ids, this);
         controller.getJobLabels(this, ids);
+        checkIfLabelsAreNotEmpty(menu, jobLabels);
+    }
+
+    private void checkIfLabelsAreNotEmpty(Menu menu, List<String> jobLabels) {
+        if (jobLabels.stream().anyMatch(String::isEmpty)) {
```

Comment:
eventually, you could use allMatch

if, among the selected jobs, all are empty, then the button can be disabled. If any of the job as a label, it will be removed and the other selected jobs will be untouched